fallout/jobs/job_types/legion/vexillarius.dm

```diff
@@ -0,0 +1,28 @@
+/datum/job/f13/legion/vexillarius
+	title = "Vexillarius"
+	total_positions = 1
+	spawn_positions = 1
+	supervisors = "The Centurion"
+	description = "Assist the higher ranks in handling your soldiers, keep them in line and focused on the goals at hand."
+	forbid = "Caesar's Legion forbids: Chem usage, Over-reliance on technology of all forms."
+
+	outfit = /datum/outfit/job/f13/legion/vexillarius
+
+	display_order = JOB_DISPLAY_ORDER_F13_VEXILLARIUS
+
+/datum/outfit/job/f13/legion/vexillarius
+	name = "_Legion Vexillarius"
+	jobtype = /datum/job/f13/legion/vexillarius
+
+	head = /obj/item/clothing/head/helmet/fallout/legion/vexillarius
+
+/datum/outfit/job/f13/legion/vexillarius/pre_equip(mob/living/carbon/human/H)
+	..()
+	glasses = /obj/item/clothing/glasses/sunglasses/big
+	back = /obj/item/storage/backpack/satchel/leather
+	uniform = /obj/item/clothing/under/fallout/legion/fatigues/brownpadded
+	shoes = /obj/item/clothing/shoes/fallout/military/legiontanboots
+	suit = /obj/item/clothing/suit/armor/fallout/legion/vexillarius
+	gloves = /obj/item/clothing/gloves/fallout/tanwraps
+	suit_store = /obj/item/gun/ballistic/automatic/fallout/smg9mm
```

Comment:
I am pretty sure this is not meant to fit in a pocket... put it and the ammo in backpack contents.